src/components/NoFundsForNetworkFee.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="notification-content">
+    <div v-if="!receiveFeeProblem && isSameChain" class="notification-text">
+      Not enough {{ assetChain }} to cover fees. Lower speed to account for gas or get more.
+    </div>
+    <div v-if="receiveFeeProblem || !isSameChain" class="notification-text">
+      Lower speed to account for gas to cover both sides of transaction, or get more.
+    </div>
+    <br />
+    <div v-if="isSameChain || receiveFeeProblem">
+      <router-link :to="accountUrl" class="btn btn-option get-bridgeAsset-btn">
+        Get {{ assetChain }}
+      </router-link>
+    </div>
+    <div v-else-if="!isSameChain">
+      <router-link :to="accountUrl" class="btn btn-option get-bridgeAsset-btn">
+        Get {{ assetChain }}
+      </router-link>
+      and
+      <router-link :to="toAccountUrl" class="btn btn-option get-bridgeAsset-btn">
+        Get {{ toAssetChain }}
+      </router-link>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    accountId: String,
+    assetChain: String,
+    receiveFeeProblem: Boolean,
+    toAssetChain: String
+  },
+  computed: {
+    accountUrl() {
+      return `/accounts/${this.accountId}/${this.assetChain}/receive`
+    },
+    toAccountUrl() {
+      return `/accounts/${this.accountId}/${this.toAssetChain}/receive`
+    },
+    isSameChain() {
+      return this.assetChain === this.toAssetChain
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.notification-content {
+  // display: flex;
+  justify-content: space-between;
+  align-items: center;
+  .get-bridgeAsset-btn {
+    max-width: 120px;
+    padding: 0 0.5rem;
+    display: inline-block;
+  }
+  .notification-text {
```

Comment:
i think we can remove this block with styles

Reply:
Maybe I can remove display:inline-block, but the padding I need, because there was a UI issue linked to that